test/unit/lodash.spec.js

```diff
@@ -0,0 +1,508 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var _ = require('lodash');
+var datasets = require('../../app/datasets');
+var Guy = require('./Guy');
+var chance = require('chance').Chance();
+
+describe('Lodash training', function ()
+{
+    describe('Math functions', function () {
+        describe('add', function () {
+
+            var params, augend, addend;
+
+            before(function () {
+                params = datasets.add();
+                augend = params[0];
+                addend = params[1];
+            });
+
+            it('should verify params', function () {
+                expect(params).to.have.length(2);
+                expect(typeof augend).to.eql('number');
+                expect(typeof addend).to.eql('number');
+                expect(addend < 0).to.eql(true);
+            });
+            
+            it('should return total', function () {
+                expect(_.add.apply(_, datasets.add(params))).to.eql(-2);
+            });
+        });
+
+        describe.only('ceil', function () {
+
+            var params, number, precision;
+
+            before(function () {
+                params = datasets.ceil();
+                number = params[0];
+                precision = params[1];
+            });
+
+            it('should verify params', function () {
+                expect(params).to.have.length(2);
+                expect(typeof number).to.eql('number');
+                expect(typeof precision).to.eql('number');
+
+                expect(Number.isInteger(number)).to.not.eql(true);
+                expect(precision).to.not.eql(0);
+            });
+
+            it('should return round up number', function () {
+                expect(_.ceil.apply(_, datasets.ceil(params))).to.eql(1.85);
+            });
+        });
+
+        describe('divide', function () {
+
+            var params, dividend, divisor;
+
+            before(function () {
+                params = datasets.divide();
+                dividend = params[0];
+                divisor = params[1];
+            });
+
+            it('should verify params', function () {
+                expect(typeof dividend).to.eql('number');
+                expect(typeof divisor).to.eql('number');
+                expect(dividend).to.not.eql(Infinity);
+
+                expect(params).to.have.length(2);
+            });
+
+            it('should divide two numbers', function () {
+                expect(_.divide.apply(_, datasets.divide(params))).to.eql(Infinity);
```

Comment:
I try to run divide function with **[ 13, 0 ]** arguments but I get error (regardless of the arguments)
```
TypeError: Cannot read property 'apply' of undefined
    at Context.<anonymous> (test/unit/lodash.spec.js:91:32)
```